File: bin/salesman_euclid_brute_force.py
#!/usr/bin/env python3

import sys
import os

bin_path = os.path.dirname(os.path.realpath(__file__))
pkgs_path = os.path.realpath(os.path.join(bin_path, '..', 'pkgs', 'python'))

sys.path.insert(0, pkgs_path)

from opticontest.salesman_euclid.brute_force import Algorithm

count = int(input())

points = []

for x in range(0, count):
    points.append(list(map(float, input().split())))

algorithm = Algorithm(points)

print(' '.join(map(str, algorithm.solution.indices)))

File: bin/salesman_euclid_brute_force.rb
#!/usr/bin/env ruby
# frozen_string_literal: true

lib = File.expand_path(File.join('..', 'pkgs', 'ruby', 'lib'), __dir__).freeze
$LOAD_PATH.unshift lib unless $LOAD_PATH.include? lib

require 'opticontest/salesman_euclid/common'
require 'opticontest/salesman_euclid/brute_force'

points = Opticontest::SalesmanEuclid::Common::Points.new

Integer($stdin.gets).times do
  points.add(*$stdin.gets.split.map(&method(:Float)))
end

algorithm = Opticontest::SalesmanEuclid::BruteForce::Algorithm.new points

puts algorithm.solution.indices.join ' '
